curriculum/challenges/_meta/rosetta-code/meta.json

```diff
@@ -649,8 +649,4 @@
       "594810f028c0303b75339ad8",
       "Zig-zag matrix"
     ]
-  ],
-  "helpRoom": "",
-  "nChallenges": 437,
```

Comment:
Was this intentional?

Reply:
Nope. Regex mistake on my part, but I suspect that can go too.

Reply:
It's only declared here and used nowhere, so I'm happy with it going away.